spec/ConvertGobstonesAstToMulangAstSpec.hs

```diff
@@ -0,0 +1,149 @@
+module ConvertGobstonesAstToMulangAstSpec (spec) where
+
+import	Test.Hspec
+import	Language.Mulang
+import	Language.Mulang.Parsers.Gobstones
+
+spec :: Spec
+spec = do
+
+describe "translateProgramGobstonesToMulangExpression" $ do
+    it "translate simple program Gobstones" $ do
+      let gobstonesAst =  parseGobstones "[{\"alias\": \"program\",\"body\": null,\"from\": 0}]"
+
+      gobstonesAst `shouldBe` MuNull
+
+    it "translate null program Gobstones" $ do
+      let gobstonesAst =  parseGobstones "null"
+
+      gobstonesAst `shouldBe` MuNull
+
+    it "translate simple procedure Call" $ do
+      let gobstonesAst =  parseGobstones "[{\"alias\": \"program\",\"body\": [{\"alias\": \"ProcedureCall\",\"from\": 14,\"to\": 18,\"arity\": \"routine\",\"name\": \"F\",\"parameters\": []}],\"from\": 0}]"
+      									
+      gobstonesAst `shouldBe` Sequence [Application (Variable "F") []]
+     
+    it "translate simple procedure declaration " $ do
+      let gobstonesAst =  parseGobstones  "[{\"alias\": \"procedureDeclaration\",\"body\": null,\"from\": 1,\"row\": 1,\"to\": 12,\"value\": \"F\",\"arity\": \"routine\",\"reserved\": false,\"led\": null,\"lbp\": 0,\"name\": \"F\",\"parameters\": []}]"
```

Comment:
Instead of adding dozens of let's and asserts in the same test, I'd split into several tests. 

A test should have only a single assertion.